nesta/core/scripts/docker_recipes/py36_amzn1_image/launch.sh

```diff
@@ -26,6 +26,7 @@ ls
 # Remove the following from reqs since they're huge
 sed -i '/luigi/d' requirements.txt
```

Comment:
I would say not worth the effort *because* in new daps, the `requirements.txt` is always relative a specified `flow`, and so bloaty reqs should go in their local repo rather than clogging it up for everyone else. This kind of stuff was one of the motivations for factoring out mono-repo reqs into a per-project basis